tests/lib/rules/space-between-siblings.js

```diff
@@ -0,0 +1,384 @@
+/**
+ * @author *****your name*****
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/space-between-siblings')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('space-between-siblings', rule, {
+  valid: [
+    {
+      filename: 'test.vue',
+      code: `
+      <template>
+        <div>
+          <ul>
+            <li>
+            </li>
+
+            <li>
+            </li>
+
+            <li>
+            </li>
+          </ul>
+        </div>
+      </template>
```

Comment:
Maybe the code snippets in the examples can be a bit shorter? It's quite noisy to read through all of them.

I think this should also suffice (for this test case):

```suggestion
      <template>
        <div>
          <div></div>

          <div>
          </div>

          <div />
        </div>
      </template>
```